package.json

```diff
@@ -16,10 +16,6 @@
   "engines": {
     "vscode": "^1.86.0"
   },
-  "activationEvents": [
-    "onLanguage:catala_en",
-    "onLanguage:catala_fr"
-  ],
```

Comment:
Why remove that ? Isn't there a chance the package will fail to activate when editing Catala files?

Reply:
This triggers a warning in VScode because those are already referenced by these config entries:
```json
    "languages": [
      {
        "id": "catala_en",
        "aliases": [
          "Catala (English)"
        ],
        "extensions": [
          ".catala_en"
        ],
        "configuration": "./language-configuration.json"
      },
      {
        "id": "catala_fr",
        "aliases": [
          "Catala (French)"
        ],
        "extensions": [
          ".catala_fr"
        ],
        "configuration": "./language-configuration.json"
      }
```

Reply:
![image](https://github.com/user-attachments/assets/de32459a-7a29-4801-b794-3b8500d004f1)

Reply:
wouahou much technology, they even have a lintfor VSCode package configurations